analysis/tbcpriest/src/SPELLS.ts

```diff
@@ -0,0 +1,94 @@
+const spells = {
```

Comment:
Hey, real nice that you're colocating your spells in your spec folder, but you no longer should have to specify the name/icon for each spell. Just an id should be sufficient, the rest of the info is either pulled from the log or from the WCL API. Less work for us! See tbc-hunter for an example.

Reply:
How do you get a reference to the icon? I'm used to the SPELLS.SPELL_NAME.icon bit for adding icons to analyzers.

Reply:
If you use the SpellIcon component it will automatically fetch the icon for the spell.